From 4ed0d7e2bc5de16135131365f713df45352a37cf Mon Sep 17 00:00:00 2001
From: Larisa Grigore <larisa.grigore@nxp.com>
Date: Wed, 14 Apr 2021 10:07:54 +0300
Subject: [PATCH 11/14] clk: s32gen1: Update SDHC clock frequency to 400MHz

The frequency was already updated in U-Boot and Linux.

Upstream-Status: Pending

Issue: ALB-6828
Signed-off-by: Larisa Grigore <larisa.grigore@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 include/dt-bindings/clock/s32gen1-clock-freq.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/dt-bindings/clock/s32gen1-clock-freq.h b/include/dt-bindings/clock/s32gen1-clock-freq.h
index 2a5afee8b..96f4d3afc 100644
--- a/include/dt-bindings/clock/s32gen1-clock-freq.h
+++ b/include/dt-bindings/clock/s32gen1-clock-freq.h
@@ -12,7 +12,7 @@
 #define S32GEN1_FXOSC_FREQ			(40 * MHZ)
 #define S32GEN1_XBAR_2X_FREQ			(800 * MHZ)
 #define S32GEN1_LIN_BAUD_CLK_FREQ		(125 * MHZ)
-#define S32GEN1_SDHC_CLK_FREQ			(200 * MHZ)
+#define S32GEN1_SDHC_CLK_FREQ			(400 * MHZ)
 
 #define S32GEN1_A53_MAX_FREQ			(1000 * MHZ)
 #define S32GEN1_ARM_PLL_VCO_MAX_FREQ		(2000 * MHZ)
-- 
2.25.1

